{
  return Exceptions.builder(Classes.class).exception(ex).unknownException("An unexpected error: ").context(errFstring,formatArgs).append(" because of: ").build();
}
